structure_tests/types/v1/structure.go

```diff
@@ -32,9 +32,20 @@ type StructureTest struct {
 	LicenseTests       []LicenseTest
 }
 
+// TODO(nkubala): remove this in favor of struct-local driver
+var Driver drivers.Driver
+
+func (st StructureTest) SetDriver(driver drivers.Driver) {
+	Driver = driver
+}
+
+func (st StructureTest) GetDriver() drivers.Driver {
+	return Driver
+}
+
 func (st StructureTest) RunAll(t *testing.T) int {
-	originalVars := utils.SetEnvVars(t, st.GlobalEnvVars)
-	defer utils.ResetEnvVars(t, originalVars)
+	originalVars := st.GetDriver().SetEnvVars(t, st.GlobalEnvVars)
```

Comment:
A lot of calls to st.GetDriver(), maybe just store it in a var?

Reply:
yeah good call, I wanted to try and make it a struct field originally but it didn't work properly. updated